_unittest/test_14_AedtLogger.py

```diff
@@ -157,12 +158,94 @@ def test_03_stdout_with_app_filter(self):
         assert "pyaedt warning: Warning for Global" in capture.content
         assert "pyaedt error: Error for Global" in capture.content
 
-        # for handler in logger.handlers:
-        #     handler.close()
-        # for handler in project_logger.handlers:
-        #     handler.close()
-        # for handler in design_logger.handlers:
-        #     handler.close()
+    def test_04_disable_output_file_handler(self):
+        content = None
+        temp_dir = tempfile.gettempdir()
+        path = os.path.join(temp_dir, "test04.txt")
```

Comment:
where is this file removed?

have you considered [NamedTemporaryFile](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) ?

Reply:
I have to use `gettempdir` because otherwise with `NamedTemporaryFile`, I cannot read the file latter on because it is deleted as soon as I closed it.
The temp dir and all its content will be deleted at the end of the test to be sure that no handlers is preventing from this operation. Thanks for the reminder.